prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -1,24 +1,49 @@
 """Redfish collector."""
 from logging import getLogger
-from typing import Any, Dict, List, Optional
+from typing import Any, Callable, Dict, List, Optional
 
 import redfish
 import redfish_utilities
-from redfish.rest.v1 import HttpClient, InvalidCredentialsError, SessionCreationError
+from cachetools.func import ttl_cache
+from redfish.rest.v1 import (
+    HttpClient,
+    InvalidCredentialsError,
+    RetriesExhaustedError,
+    SessionCreationError,
+)
 
 logger = getLogger(__name__)
 
 
+# pylint: disable=R0903
+# pylint: disable=R0913
```

Comment:
not-blocking: I think you can use `R0903, R0913`

Reply:
can you use extended version of those so we can understand without checking what it means
`# pylint: disable=too-few-public-methods`

Reply:
It also looks like I can omit `RO903` for too-few-public-methods since the `get_discover` method is added in the class now.